### PR TITLE
Add tags on cards for people section

### LINKED ISSUE #5707: UI Improvements Planning Issue : People Section
## Description
The basic Card UI of people section is already on halfway.
Now the further improvements are left.
As we were told during proposal that we should start off with easier ones first. And they should be given beginning of time in timeline. So we will start off with people section first which is the easiest.
The final UI for the people's page : 

Screen Shot 2019-05-19 at 2 41 35 AM

@gautamig54 We will add checklists here
#### card ui of users 
- [x] Vertical to horizontal cards @CleverFool77 
- [x] Add tags on cards @CleverFool77 
- [x] Add ellipsis which contains the user details as shown in the design @gautamig54 

#### location
- [x] Add user's location. @gautamig54 

#### search by
- [ ] Add search by various places option below maps. 

#### others
 - [ ] Make the map sticky. 
 - [x] position map on the top instead of bottom, in the mobile view @CleverFool77 
 - [x] Fix overflow of tags @CleverFool77

### PR DESCRIPTION
**Pull request by CleverFool77**
Fixes #5707 part

Screenshot from 2019-05-22 15-47-45
Screenshot from 2019-05-22 15-48-52

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by CleverFool77** [comment]
Hi @jywarren @gauravano I wanted to know that Is the method I followed for this is correct ? 
I need bit of help here.
Thanks !!

**Comment by jywarren** [comment]
I think this is looking good -- do you think you could make the tags a bit more like they appear on https://publiclab.org/tags? This is awesome!

File: app/views/users/list.html.erb [hunk 1]
```diff
@@ -38,9 +38,12 @@
               <div class="card-info">
                 <a href="/profile/<%= user.username %>"><%= user.username %></a>
                   <div class="sub-info">
-                    <div>Last Activity : <%= t = user.revisions.order(timestamp: :desc).first.try(:created_at);time_ago_in_words(t) if (t) %></div>
-                    <div>History : <%= user.nodes.count %> <%= t('users.list.notes_and_edits') %></div>
-                    <div> Joined : <%= distance_of_time_in_words(user.created_at, Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %></div>
+                  <span id="tags">
+                  <% Tag.sort_according_to_followers(user.tags,"desc") %>
```
Comment (jywarren) [inline app/views/users/list.html.erb]: whoa this is cool! is this a Rails method?

File: app/views/users/list.html.erb [hunk 2]
```diff
@@ -38,9 +38,12 @@
...
+                  <% user.tags.limit(3).each do |tag| %>
```
Comment (jywarren) [inline app/views/users/list.html.erb]: Should we store `Tag.sort_according_to_followers` in a var like: 

```ruby
tags = Tag.sort_according_to_followers
```
then use
```ruby
<% tags[0..2].each do |tag| %>
```

that way your sorting will persist, and also we're not running a new query?

**Comment by CleverFool77** [comment]
> I think this is looking good -- do you think you could make the tags a bit more like they appear on https://publiclab.org/tags? This is awesome!

Hi @jywarren The code I'm using for tags UI is same as used in all other ages. It's just due to my screenshot which seems to changes the color to bit purple from blue.

**Comment by CleverFool77** [comment]
I just updated the code and new UI is ; 
Screenshot from 2019-05-24 11-42-36

the color looks different but in actual its `badge-primary` only.
Thanks !!!

File: app/views/users/list.html.erb (hunk 2 above)
Comment (CleverFool77) [reply app/views/users/list.html.erb]: yes Now, the sorting is persisting.
I've updated it.
Thanks !!!

**Comment by jywarren** [comment]
Oh ok! Good to know, thanks! Is this ready for final review and merge? Thank you!

**Comment by jywarren** [comment]
Actually one more thing; i think @gautamig54 noticed this but Junction is set as the font for many parts of the page, but as per the Style Guide, we want to only use it for headings like `h1, h2, h3`. Would you mind adjusting this? Thank you so much!

**Comment by CleverFool77** [comment]
Hi @jywarren I just fixed the font bug. I removed the assignment of font is beginning for whole section. So now It's working only for headings.
Thanks !!!

**Comment by CleverFool77** [comment]
This is ready for merge. I guess.
Thanks !!!

**Comment by CleverFool77** [comment]
I'll update few more stuff.
Thanks !!

**Comment by CleverFool77** [comment]
I just noticed that the class which helps in stacking map above in smaller screen seems to be removed.
Maybe it was removed by someone whole resolving merge conflicts. 
I'll add it back then.

**Comment by CleverFool77** [comment]
Hi @jywarren I did the changes online as my local branch is mess but checked it locally too 
I guess Now this PR is ready for merge.
Thanks !!

**Comment by jywarren** [comment]
Oh cool! Can you upload one more screenshot? (hopefully screenshots won't be as needed soon - see https://github.com/publiclab/plots2/pull/5320) Just to see how the fonts work? Thank you!

**Comment by CleverFool77** [comment]
> Oh cool! Can you upload one more screenshot? (hopefully screenshots won't be as needed soon - see #5320) Just to see how the fonts work? Thank you!

Hi @jywarren
Sorry I slept yesterday before this message.
Here is the scrrenshot.
Screenshot from 2019-05-25 14-47-09

**Comment by CleverFool77** [comment]
The new feature of having screenshot during system test is awesome.
Damn !!!!!!!!!!!!!!!!!!!!!

**Comment by jywarren** [comment]
Awesome here! 😄

**Pull request merged**